src/page-view.ts

```diff
@@ -134,8 +141,11 @@ export class PageViewManager {
     }
 
     startMeasuringScrollPosition() {
-        window?.addEventListener('scroll', this._updateScrollData)
-        window?.addEventListener('scrollend', this._updateScrollData)
+        // setting the third argument to `true` means that we will receive scroll events for other scrollable elements
+        // on the page, not just the window
+        // see https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#usecapture
+        window?.addEventListener('scroll', this._updateScrollData, true)
+        window?.addEventListener('scrollend', this._updateScrollData, true)
         window?.addEventListener('resize', this._updateScrollData)
```

Comment:
trying to think defensively (so feel free to shoot this down) is it worth debouncing here?

the processing looks cheap but we'll get a lot of updates processed that we don't need 🤷